ddclient

```diff
@@ -1022,7 +1022,7 @@ sub write_cache {
     ## construct the cache file.
     my $cache = "";
     foreach my $h (sort keys %cache) {
-        my $opt = join(',', map { "$_=" . define($cache{$h}{$_},'') } sort keys %{$cache{$h}});
+        my $opt = join(',', map { "$_=" . ($cache{$h}{$_} // '') } sort keys %{$cache{$h}});
```

Comment:
```suggestion
        my $opt = join(',', map { "$_=" . $cache{$h}{$_} // '' } sort keys %{$cache{$h}});
```

Reply:
No, because `.` has higher precedence than `//`. Removing the parentheses is equivalent to:

```suggestion
        my $opt = join(',', map { ("$_=" . $cache{$h}{$_}) // '' } sort keys %{$cache{$h}});
```